### DISCUSSION TITLE
how to close this window?

### DISCUSSION THREAD

**Discussion by xiaozhe2333** [retorquere/zotero-better-bibtex#2402] [Q&A] [answered]
I'm not sure whether this window is caused by this plug-in, but I still want to ask how to close this window
![image](https://user-images.githubusercontent.com/92240019/216546621-032ad5c7-d620-44e9-a718-073a9de82825.png)

**Comment by retorquere** [comment, marked as answer]
Not a BBT window. It says "Obsidian", I'd start there.

**Reply by xiaozhe2333** [reply]
thanks，I“II try to find it
